### PR TITLE
Added AdGuard Quick Fixes filter

### PR DESCRIPTION
**Pull request by Alex-302**

### CODE REVIEW

**Review by BlazDT — COMMENTED** [review]
When adding rules there, we should also note the time of addition (due to 14 days). Else you have to check GitHub history when it was added.

Probably should be added as comment.

**Comment by Alex-302** [comment]
Vscode shows this info (maybe with GitLens extension)
image

File: QuickFixesFilter/sections/quickfixes.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+! - Temporary: All rules added to this file are temporary and must be removed after 14 days from the date of their addition.
+! - Flexibility: Rules can be added without restrictions related to the source of the issue, ensuring quick
+!   response to solving important problems.
+!
+
+
+
+
+
+
```
Comment (slavaleleka) [inline QuickFixesFilter/sections/quickfixes.txt:21]: too many empty lines

**Review by vozersky — APPROVED** [review]

**Review by ameshkov — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Review by AdamWr — APPROVED** [review]

File: QuickFixesFilter/sections/quickfixes.txt [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+! solution to ensure that users of the AdGuard Mv3 extension experience minimal disruption while waiting for
+! the extension to update along with the static filters.
+!
+! Key points:
+! - Purpose: This file is used to store temporary rules to address critical issues regardless of which filter is
+!   causing the problem or the type of problem (e.g., missed ads).
+! - Limited lifetime: All rules added to this file are temporary will be removed 14 days after they are added.
+! - Flexibility: Rules can be added to resolve critical issues quickly, with no restrictions based on the source of the problem.
+!
+! Rules must be added with the date of addition for ease of future deletion.
```
Comment (sonya-orlova) [inline QuickFixesFilter/sections/quickfixes.txt:15]: вот так бы сделала
To make it easier to delete rules in the future, they must be added with their date of addition.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -226,3 +226,10 @@ a task comment or hints, put them next to the same structure in the file.
 
 - Purpose: this filter blocks various kinds of ads on Turkish-language sites.
 - [Turkish folder](https://github.com/AdguardTeam/AdguardFilters/tree/master/TurkishFilter/sections)
+
+### AdGuard Quick Fixes filter
+
+- Purpose: This filter serves as a quick-response solution, ensuring that AdGuard Mv3 extension
```
[the remark this answers is not in the document]
Comment (Alex-302) [reply CONTRIBUTING.md:232]: Да норм вроде.

File: QuickFixesFilter/sections/quickfixes.txt (hunk 2 above)
Comment (Alex-302) [reply QuickFixesFilter/sections/quickfixes.txt:15]: Ок, заменил.

**Pull request merged**